pkg/controller/operators/catalog/operator.go

```diff
@@ -1172,11 +1176,13 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 		}
 	}
 
+	logger.Infof("Phase coming in: %s", plan.Status.Phase)
```

Comment:
this looks like it should be debug level or trace

Reply:
Sorry - left over debugging code!